scripts/release-bit-verification.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/sh
+
+############################################################################
+#  PREREQUISITES FOR THIS SCRIPT
+# 1. Redistributable-binary(.rpm) should be passed as the first argument
+# 2. Login to the cluster should be done prior to running this script
+# 3. The cluster should be in a state where it can be used for testing
+#
+# USAGE:
+# ./check-binary.sh redistributable-binary
+#
+# Example: ./check-binary.sh ~/Downloads/odo-redistributable-2.4.3-1.el8.x86_64.rpm
+#
+
+shout() {
+  echo "--------------------------------------------------------------------------"
+}
+# Check SHASUM for all the binary files and there should be no difference
+
+# Create a Temp directory 
+WORKING_DIR=`mktemp -d`
+
+# Extract from rpm file 
+rpm2cpio ${1} | cpio -idmvD $WORKING_DIR
+pushd $WORKING_DIR/usr/share/odo-redistributable/
+
+# Check sha256sum for all the files
+while IFS= read -r line; do
+    read -r SHA FILE <<<"$line"
+    read -r SHATOCHECK FILE <<< `sha256sum $FILE`
+    if [[ $SHA == $SHATOCHECK ]]; then
+        # Print if the file is correct
+        printf '%-50s\U0002705\n' $FILE 
+    fi
+done < SHA256_SUM
+
+shout
+
+# Copy binary for testing purpose
+OS=`uname -s`
+ARCH=`uname -m`
+
+if [[ $OS == "Linux" ]]; then
+    if [[ $ARCH == "x86_64" ]]; then
+        cp ./odo-linux-amd64 $GOBIN/odo
+    fi
+fi
+
+# Check odo verion and if it is correct 
+VERSION=`cat VERSION`
```

Comment:
Where is this VERSION file located? Don't you need to mention the path to this file?

Reply:
And if this is VERSION is same as https://github.com/redhat-developer/odo/blob/main/build/VERSION, we will need to update that file as well.

Reply:
> Where is this VERSION file located? Don't you need to mention the path to this file?

This file is located in the `.rpm` file, which is build by the brew.
This is what we get when we extract the rpm file.
```sh
/tmp/tmp.dOU5sjCuoV/usr
├── lib
└── share
    ├── licenses
    │   └── odo-redistributable
    │       └── LICENSE
    └── odo-redistributable
        ├── odo-darwin-amd64
        ├── odo-darwin-amd64.tar.gz
        ├── odo-darwin-arm64
        ├── odo-darwin-arm64.tar.gz
        ├── odo-linux-amd64
        ├── odo-linux-amd64.tar.gz
        ├── odo-linux-arm64
        ├── odo-linux-arm64.tar.gz
        ├── odo-linux-ppc64le
        ├── odo-linux-ppc64le.tar.gz
        ├── odo-linux-s390x
        ├── odo-linux-s390x.tar.gz
        ├── odo-windows-amd64.exe
        ├── odo-windows-amd64.exe.zip
        ├── SHA256_SUM
        └── VERSION
```

> And if this is VERSION is same as https://github.com/redhat-developer/odo/blob/main/build/VERSION, we will need to update that file as well.

I am not sure if it is required to update there or not.
But the content of version file inside the rpm contains the latest release verison
```
$ cat /tmp/tmp.dOU5sjCuoV/usr/share/odo-redistributable/VERSION                                          
2.4.3
```

Reply:
So, I am curious, how do I obtain the .rpm file? I did not find anything [in here](https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/) or on the GH release page..

Reply:
It's available in the errata raised for the release.